src/test/resources/org/rust/ide/intentions/fixtures/add_derive/add_derive_enum_after.rs

```diff
@@ -1,3 +1,3 @@
-    #[derive(<caret>)]enum Test {
+    #[derive(<caret>)] enum Test {
```

Comment:
We want a new line here. This should be possible to do with spacing builder. This can be fixed in another PR.